src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -173,6 +174,11 @@ public void WorkspaceActualSize(double width, double height)
             RaisePropertyChanged("WorkspaceActualWidth");
         }
 
+        /// <summary>
+        /// This property is the ViewModel that will be passed to the File Trust Warning popup when is created.
+        /// </summary>
+        public FileTrustWarningViewModel FileTrustWViewModel { get; set; }
```

Comment:
internal?

Reply:
@QilongTang changed to be internal and the 'W' character was deleted.
https://github.com/DynamoDS/Dynamo/pull/12916/commits/6e3af78f9f27e36771c3ceb54e8ea240bf3c9047